AWS/rekognition/03.rekognition_face_compare.py

```diff
@@ -0,0 +1,58 @@
+import os, sys
+import pprint, argparse
+import boto3
+
+
+def main(args):
+    rekognition = boto3.client('rekognition')
+
+    if not args.source_img or not args.target_img:
+        raise Exception('Missing argument(s): sourceimg or targetimg.')
+
+    if args.s3_bucket:
+        # use images from s3 bucket when specified
+        source_img_kwargs = {
+            'S3Object': {
+                'Bucket': args.s3_bucket,
+                'Name': args.source_img,
+            }
+        }
+        target_img_kwargs = {
+            'S3Object': {
+                'Bucket': args.s3_bucket,
+                'Name': args.target_img
+            }
+        }
+
+    else:
+        # use images from local
+        with open(args.source_img, 'rb') as f:
+            source_img = f.read()
+        
+        with open(args.target_img, 'rb') as f:
+            target_img = f.read()
+        
+        source_img_kwargs = {'Bytes': source_img}
+        target_img_kwargs = {'Bytes': target_img}
+
+    response = rekognition.compare_faces(
+        SourceImage=source_img_kwargs,
+        TargetImage=target_img_kwargs
+    )
+
+    pp = pprint.PrettyPrinter(indent=2)
+    pp.pprint(response)
+
+    return
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='Script to try rekognition:detect_label()')
```

Comment:
solid job, can include more inline comments on what the classes do. and/or descriptions on what the script does amid argparse. this will also assist during the workshop

Reply:
alright thanks man

Reply:
will do asap

Reply:
done